### ISSUE TITLE
Column not found: 1054 Unknown column 'found'

### ISSUE THREAD

**Issue by alex-mokkas** [rainlab/translate-plugin#642]
I've just installed OcroberCMS 2 and the Translate plugin.

1. I've added a second language and marked it as "Enabled" and "Default".
2. On "Translate messages" I've clicked on "Scan for messages" and the popup came up (I cannot also see the checkboxes next to the two options "Purge all messages first" and "Purge missing messages after scan").
3. Error came up.

I've checked the database table "rainlab_translate_messages" and of course the column "found" was missing. I've added it by hand and the problem was solved.

**Comment by massimomegistus** [comment]
I confirm this issue.
After 1.8.0, version.yaml get broken and the update_messages_table.php is not executed
This section
**1.6.10:**
    - Add option to purge deleted messages when scanning messages
    - Add Scan error column on Messages page
    - update_messages_table.php
    - Fix translations that were lost when clicking locale twice while holding ctrl key
    - Fix error with nested fields default locale value
    - Escape Message translate params value
now is unproperly formatted
**1.6.10:** Add option to purge deleted messages when scanning messages, Add Scan error column on Messages page, update_messages_table.php, Fix translations that were lost when clicking locale twice while holding ctrl key, Fix error with nested fields default locale value, Escape Message translate params value.

**Comment by daftspunk** [comment]
Thanks for reporting this. It is fixed in fb1a612ce3cef1f24ed5713858389c1a5041140a (v1.8.5)

To patch your installation please run the following

```
php artisan tinker
```

And paste in the following

```
if (!Schema::hasTable('rainlab_translate_messages')) { return; } if (!Schema::hasColumn('rainlab_translate_messages', 'found')) { Schema::table('rainlab_translate_messages', function($table) { $table->boolean('found')->default(1); }); } exit
```
